bin/deployment/deploy

```diff
@@ -82,18 +82,30 @@ isrpm() {
     return $?
 }
 
+update_legacy_keystore() {
+    local CP_CERT_HOME="/etc/candlepin/certs"
+    local CP_KEYSTORE="${CP_CERT_HOME}/tc_keystore.jks"
+    local CP_KEYSTORE_PWFILE="${CP_CERT_HOME}/tc_keystore_pass.txt"
 
-update_keystore() {
-    if [ ! -f $CONTAINER_CONF_DIR/keystore ]; then
+    local CONTAINER_KEYSTORE="${CONTAINER_CONF_DIR}/keystore"
+
+    if [ ! -f $CP_KEYSTORE ]; then
+        info_msg "Legacy keystore does not exist; skipping symlink creation/update"
+        return 0 # nothing to do -- legacy keystore hasn't been generated
+    fi
+
+    if [ ! -f $CONTAINER_KEYSTORE ]; then
         info_msg "Symlinking keystore for tomcat"
-        $SUDO ln -s /etc/candlepin/certs/keystore $CONTAINER_CONF_DIR/keystore
+        $SUDO ln -s $CP_KEYSTORE $CONTAINER_KEYSTORE
```

Comment:
Running the deploy script on the Centos Stream 8 vm fails with:
```
...
Generating certificates...
Done. New certificate generated: /etc/candlepin/certs/candlepin-ca.crt
Generating legacy JKS keystore...
Certificate was added to keystore
Done. Keystore generated: /etc/candlepin/certs/tc_keystore.jks
Symlinking keystore for tomcat
ln: failed to create symbolic link '/etc/tomcat/keystore': File exists
```
And every time after the initial it still fails with:
```
...
Certificates already generated; using existing certs
Symlinking keystore for tomcat
ln: failed to create symbolic link '/etc/tomcat/keystore': File exists
```

Reply:
\*grumble\*
I fixed this once already. Hopefully I just accidentally reverted something. Looking into it.

Reply:
nevermind, this was just me having deleted the /etc/candlepin/certs (to test with a 'fresh' checkout) directory without actually deleting the /etc/tomcat/keystore symlink. When I deleted both the dirt and symlink then it worked